cluster-autoscaler/cloudprovider/kubemark/kubemark_linux.go

```diff
@@ -1,3 +1,5 @@
+// +build linux
```

Comment:
As discussed offline, can't be moved, only one line comments can go before the build directive.